acceptance.sh

```diff
@@ -155,6 +155,6 @@ cd ${DIR}/acceptance
 sudo GOPATH=${GOPATH} PATH=${PATH} ./startMockAgents.sh --no-wait
 
 # launch cucumber/capybara with colorized output disabled for better readability in Jenkins
-CUCUMBER_OPTS=--no-color ./runUIAcceptance.sh -a https://${HOSTNAME} $*
+CUCUMBER_OPTS=--no-color ./runUIAcceptance.sh -a https://${HOSTNAME} --tags ~@skip $*
```

Comment:
If you add PENDING to the Given statement, then you don't need to specify any extra arguments here